BESTPRACTICES.md

```diff
@@ -0,0 +1,18 @@
+# Best Practices
+---
+## Weave 
+> Is weave.Metadata metadata always required?
+
+This allows you to upgrade the code on chain without downtime
+
+> What is the best practices for using proto value types on modeling currency amount, IDs, timestamp?  
+
+ - Currency example [weave/x/coin](https://github.com/iov-one/weave/tree/master/coin)
+ - For incremetal ID use `orm.IDGenBucket`
```

Comment:
We can use new `morm/ModelBucket` now :)

I'd be happy for your feedback on that

Reply:
Read [iov-one/weave#712](https://github.com/iov-one/weave/issues/712). I am delighted to raise new concerns for you guys :relaxed: I will add a _TODO_ line until ```morm/ModelBucket``` merged into weave. Want to keep this document unified for future newcomers.